### PR TITLE
mkcloud: correct+extend lvm config examples

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -394,18 +394,19 @@ function onhost_create_cloud_lvm()
     fi
 
     echo "Checking for LVs treated by LVM as valid PV devices ..."
-    if lvmdiskscan | egrep "/dev/($cloud/|mapper/$cloud-)"; then
+    if lvmdiskscan | egrep "/dev/($cloudvg/|mapper/$cloudvg-)"; then
```
Comment (aspiers) [inline scripts/mkcloud]: Yes, thanks and sorry for getting that wrong :)

File: scripts/mkcloud [hunk 2]
```diff
@@ -394,18 +394,19 @@ function onhost_create_cloud_lvm()
...
         error=$(cat <<EOF
 Error: your lvm.conf is not filtering out mkcloud LVs.
 Please fix by adding the following regular expressions
 to the filter value in the devices { } block within your
 /etc/lvm/lvm.conf file:
 
-    "r|/dev/mapper/$cloud.*|", "r|/dev/$cloud/.*|"
+    "r|/dev/mapper/$cloudvg-|", "r|/dev/$cloudvg/|", "r|/dev/disk/by-id/|"
```
Comment (aspiers) [inline scripts/mkcloud]: Why `/dev/disk/by-id/`?  Couldn't that filter out lots of devices which are unrelated to `mkcloud`?

File: scripts/mkcloud [hunk 3]
```diff
@@ -394,18 +394,19 @@ function onhost_create_cloud_lvm()
...
 
-The filter should also include something like "r|/dev/dm-1[56]|", but
+The filter should also include something like "r|/dev/dm-|" or "r|/dev/dm-1[56]|", but
```
Comment (aspiers) [inline scripts/mkcloud]: Similarly to `/dev/disk/by-id/`, I'm worried that filtering `/dev/dm-` would exclude other important devices.

**Comment by jdsn** [comment]
:+1:

File: scripts/mkcloud (hunk 2 above)
Comment (bmwiedemann) [inline scripts/mkcloud]: yes, but those other devices also appear elsewhere under other names. e.g. /dev/sda2

File: scripts/mkcloud (hunk 3 above)
Comment (bmwiedemann) [inline scripts/mkcloud]: It could indeed, which is why such exceptions are now mentioned 2 lines below

**Comment by MaximilianMeister** [comment]
:+1:

**Pull request merged**